stglib/tests/test_scripts.py

```diff
@@ -6,7 +6,7 @@ def exo_raw(glob_att, config_yaml):
     result = subprocess.run(
         ["python", "../../../scripts/runexocsv2cdf.py", glob_att, config_yaml],
         capture_output=True,
-        cwd="stglib/tests/data",
```

Comment:
This path (and everywhere else it is specified) needs to stay this way for automated testing. For local testing, run purest from the root stglib directory so this will work. So `cwd="stglib/tests/data",` needs to be specified everywhere.

Reply:
fixed path back to cwd="stglib/tests/data".  Note: there is a line in the stglib Documentation in sec 15.5.2 that is confusing "pytest can be run from within the tests directory." which contributed to me causing this issue. Suggest re-wording that line in the docs.